integration-tests/ci-visibility.spec.js

```diff
@@ -61,36 +64,15 @@ const testFrameworks = [
     runTestsWithCoverageCommand: './node_modules/nyc/bin/nyc.js -r=text-summary node ./ci-visibility/run-mocha.js',
     type: 'commonJS'
   },
-  {
-    ...mochaCommonOptions,
-    testFile: 'ci-visibility/run-mocha.mjs',
-    dependencies: ['mocha', 'chai@v4', 'nyc', '@istanbuljs/esm-loader-hook'],
-    expectedCoverageFiles: [
-      'ci-visibility/run-mocha.mjs',
-      'ci-visibility/test/sum.js',
-      'ci-visibility/test/ci-visibility-test.js',
-      'ci-visibility/test/ci-visibility-test-2.js'
-    ],
-    runTestsWithCoverageCommand:
-      `./node_modules/nyc/bin/nyc.js -r=text-summary ` +
-      `node --loader=./node_modules/@istanbuljs/esm-loader-hook/index.js ` +
-      `--loader=${hookFile} ./ci-visibility/run-mocha.mjs`,
-    type: 'esm'
-  },
   {
     ...jestCommonOptions,
     testFile: 'ci-visibility/run-jest.js',
     runTestsWithCoverageCommand: 'node ./ci-visibility/run-jest.js',
     type: 'commonJS'
-  },
-  {
-    ...jestCommonOptions,
-    testFile: 'ci-visibility/run-jest.mjs',
-    runTestsWithCoverageCommand: `node --loader=${hookFile} ./ci-visibility/run-jest.mjs`,
-    type: 'esm'
```

Comment:
esm tests were being skipped altogether. Until we add them back this is noise, so I'm removing it